data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
-	spaceport `The Spaceport is made up of multiple large, hexagonal landing areas surrounding a spaceport complex with an air control tower, a bar and multiple entrances to a large underground hangar used by civilians. The spaceport seems to be very busy at all times, but doesn't appear to be going over it's capacity.`
-	spaceport `	Outside of the main area is a relatively small Alteran military base, following a smaller hexagonal pattern, with a larger central complex with three arms that extend into the surrounding landing pads. The spaceport itself is located in the middle of the imperial capital city, not too far off from the palace.`
+	description `The Alteran capital world of Celestis is one of the most densely populated Alteran worlds, with over 12 billion inhabitants before the war. Its proportions seem to be fairly similar to those of the Earth, it consists of highly populated futuristic cities full of towering skyscrapers, a few bigger mountain ranges and large oceans. Unlike Earth though, it is not overpopulated and surprisingly, the oceans appear to be in pristine condition.`
+	description `	Even though the Alteran Empire is a monarchy, the seat of the government does not stand out and is very down to earth on closer inspection. This corresponds to the respect that the Alterans have for the monarchy, which does not rule by fear, but by a mutual understanding, teamwork and the meritocratic ways of the government.`
+	spaceport `The Spaceport is made up of multiple large, hexagonal landing areas surrounding a spaceport complex with an air control tower, a bar and multiple entrances to a large underground hangar used by civilians. Though the spaceport is very busy at all times, it is not in danger of exceeding its capacity.`
+	spaceport `	Outside of the main area is a relatively small Alteran military base, laid out in a smaller hexagonal pattern, with a larger central complex with three arms that extend into the surrounding landing pads. The spaceport itself is located in the middle of the imperial capital city, not too far off from the palace.`
 	shipyard "Ancient Advanced"
 	outfitter "Ancient Advanced"
 
 planet Dakara
 	landscape land/badlands0
-	description `Dakara is a very rocky and sandy world, used almost exlusively for mining purposes. It used to be far greener, but ended up a bit too close to the sun, resulting in higher temperatures and little rainfall. Most of the long-term population, dating back generations, has a darker skintone than the usual, almost white, Alteran skintone.`
-	description `	The buildings have few windows and large air-conditioners on their roofs, as well as a garage, to reduce the heat and the potential damage done by it. To take it a step further, whoever designed the initial building strategy, appears to have thought of everything, as even the tiniest building is connected to a network by either an overground glass tunnel or an underground path.`
```

Comment:
Putting a garage above the house only makes sense if it protects against something coming from above. That's why I referred to the sun. The later edit is again removing the speculation.